beacon-chain/casper/validator.go

```diff
@@ -205,9 +205,9 @@ func VotedBalanceInAttestation(validators []*pb.ValidatorRecord, indices []uint3
 	// find the total and vote balance of the shard committee.
 	var totalBalance uint64
 	var voteBalance uint64
-	for _, attesterIndex := range indices {
+	for index, attesterIndex := range indices {
 		// find balance of validators who voted.
-		bitCheck, err := bitutil.CheckBit(attestation.AttesterBitfield, int(attesterIndex))
+		bitCheck, err := bitutil.CheckBit(attestation.AttesterBitfield, index)
```

Comment:
This is a bug. The bitfield represents the indices in relation to the committee, not the active validator set.